.envrc

```diff
@@ -159,57 +141,50 @@ done
 
 ### Git ###
 
-info "Configuring git..."
+debug "Configuring git..."
```

Comment:
Moving all `info` messages to `debug` and we do not exit direnv unsuccessfully (that way we always stay within the .venv if activated).

Reply:
I liked these, why remove them? It helped to see if a part of direnv was going slow. Now it's just a black box.

Reply:
I come more from the angle of output the least possible unless you're debugging.
As I mentioned on Slack, I've added `SENTRY_DIRENV_DEBUG` in order to add them back for whoever wants it.